src/flag_gems/runtime/config.py

```diff
@@ -0,0 +1,115 @@
+import copy
+
+import triton
+
+from . import backend
+from .device import device
+
+
+class Config:
```

Comment:
I don't see the point of this class.

1. It is actually not a config. It is a config loader.